WebService/appsettings.ini

```diff
@@ -19,6 +19,9 @@
 # TCP port where to listen for web service requests. 9003 by default.
 webservice_port = 9003
 
+# Template (name) to be used for solution seeding
```

Comment:
add comment about the folder where this is found, and an example, e.g. "foo" => "foo.json"